.pre-commit-config.yaml

```diff
@@ -0,0 +1,44 @@
+exclude: "docs|.git|geometric_data"
+default_stages: [pre-commit]
+fail_fast: true
+
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v5.0.0
+    hooks:
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+
+  # Can run individually with `pre-commit run isort --all-files`
+  - repo: https://github.com/PyCQA/isort
+    rev: 5.13.2
+    hooks:
+      - id: isort
+        exclude: "geometric_features/__init__.py"
```

Comment:
@xylar isort reordering the imports for this file broke everything, so I've excluded it here.

Reply:
Let's follow up on that in a separate PR.  We don't need all the `__future__` stuff anymore.  That was for python 2.  Maybe that will help. But if that's not the problem, we maybe need to add missing imports somewhere else.